client/js/components/GameView/Card.vue

```diff
@@ -198,6 +214,29 @@ export default {
   }
 }
 
+.scuttled-by-overlay {
+  height: 80%;
+  top: -32px;
+  transition: all 1s ease;
+}
+.slide-below-leave-active,
+.slide-above-leave-active,
+.in-below-out-left-leave-active {
+  position: absolute;
+}
+// slide-below (enter and leave below)
+.slide-below-enter,
+.slide-below-leave-to {
+  opacity: 0;
+  transform: translateY(32px);
+}
+// slide-above (enter and leave above)
+.slide-above-enter,
+.slide-above-leave-to {
+  opacity: 0;
+  transform: translateY(-32px);
+}
```

Comment:
These are copied from GameView. It would be better I think to pull these into scss mixins we can include in both places. Okay for the moment imo

Reply:
File a follow up :), I agree we should dry this up. If it's easy to include might be worth doing here and not rushing this feature out the door. Do we have anywhere we store global mixins that are shared?

Reply:
Currently we have no scss mixins or reused css. The closes atm is global styles that are applied in the unscoped App.vue style block. This would likely be the first reused scss stuff so I'm thinking new issue

Reply:
https://github.com/cuttle-cards/cuttle/issues/232